.github/ISSUE_TEMPLATE.md

```diff
@@ -21,6 +21,8 @@ _3._
 
 #### Package log
 _Check Package Center or `/usr/local/{package}/var/`_
+
+_DSM7: `/var/log/packages/{package}.log`_
 ```
```

Comment:
I have integrated this in the official issue templates under in the community profile under `Insights - Community - issue templates`.
The ISSUE_TEMPLATE.md is now obsolete (on master branch).

I propose to incorporate any other DSM7 related changes of the templates on the master branch.

Reply:
Thanks I've removed the commit